shopify-eventbridge-lambda/shopify-eventbridge-lambda/README.md

```diff
@@ -0,0 +1,114 @@
+# Shopify Amazon EventBridge Partner Integration to AWS Lambda
+
+This pattern demonstrates how to use the Shopify Amazon EventBridge SaaS integration and AWS Lambda to process events from Shopify. This pattern is leveraging the Shopify Amazon EventBridge webhook integration to send product updated events from the customer's Shopify account to their AWS account, via an Amazon EventBridge Partner event bus. Once the Shopify events are in the customer's account, an Amazon EventBridge rule routes updated product events to a downstream Lambda function. In production cases, the Lambda function could transform the event, send it to a downstream application, archive it in a warehouse service, or send a notification email to a customer using SES. Amazon CloudWatch Log Groups are provisioned for debugging and auditing. This pattern deploys two EventBridge rules, one Lambda function, and two CloudWatch Log Groups.
+    
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/shopify-eventbridge-lambda
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS CDK CLI](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) (AWS CDK) installed
+* [Create a Shopify Account](https://accounts.shopify.com/signup?rid=645a3a17-4ed1-4d68-94ab-de416384059a) if you do not already have one and log in. 
+* [Set up the Shopify Amazon EventBridge webhook integration](https://shopify.dev/docs/apps/build/webhooks/configuration/eventbridge) if you have not already configured the integration.
+
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd shopify-eventbridge-lambda
+    ```
+3. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the app.py file. A command-line argument is needed to deploy the CDK stack, "shopifyEventBusName". This argument should be the name of the **SaaS event bus** associated with your Shopify [partner event source](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-saas.html).
```

Comment:
I believe this command needs to be run from within `serverless-patterns/shopify-eventbridge-lambda/shopify-eventbridge-lambda`. 

Can you move contents of the internal folder ```shopify-eventbridge-lambda``` outside and remove the internal folder if not needed?